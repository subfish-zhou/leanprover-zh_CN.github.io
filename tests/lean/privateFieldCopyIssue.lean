structure A where
  a : Nat
  protected b : Nat
  private c : Nat

structure B where
  a : Nat
  private d : Nat
  protected e : Nat

structure C extends A, B

#print A.b
#print A.c
#print C.d
#print C.e
