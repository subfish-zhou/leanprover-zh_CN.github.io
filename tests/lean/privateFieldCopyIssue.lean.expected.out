protected def A.b : A → Nat :=
fun self => self.2
private def A.c : A → Nat :=
fun self => self.3
private def C.d : C → Nat :=
fun self => self.2
protected def C.e : C → Nat :=
fun self => self.3
